src/App.js

```diff
@@ -3,20 +3,18 @@ import LifeChartHeader from "./components/header/LifeChartHeader"
 import "./i18n.ts";
 import Footer from "./components/footer/Footer";
 import {NavigationContextProvider} from "./context/navigationContext/NavigationContext";
-import {InputContextProvider} from "./context/userinputs/InputContext";
```

Comment:
We need this context to track the inputs made by the user over the whole application